src/SpecFile.cpp

```diff
@@ -4494,6 +4504,14 @@ bool SpecFile::load_file( const std::string &filename,
         }//if( orig_file_ending=="xml" )
 
 
+        if( orig_file_ending=="xml" || orig_file_ending=="rco")
```

Comment:
RadiaCode exports as `.xml` by default, '.rco' is my local convention to indicate that this XML file is radiacode data and not ... some other thing.